### DISCUSSION TITLE
KeyASIO GUI

### DISCUSSION THREAD

**Discussion by miyuki-ts** [Milkitic/KeyASIO.Net#18] [Q&A]
This is my settings:

![image](https://github.com/Milkitic/KeyASIO.Net/assets/93956624/87c87d80-da7e-4dcc-9745-d77f662aac43)
![image](https://github.com/Milkitic/KeyASIO.Net/assets/93956624/5e86e5c2-8650-4431-81b5-260663334c7f)

how do i make it sound as normal osu! for effects ingame, other question what is Balance Factor, it is default on 30% and my master volume is 30% which i set it to and music on 60%

**Comment by miyuki-ts** [comment]
i just want to make sure, i understand the settings and set them correct, because i really love this software so thank you for the great work.

**Comment by Milkitic** [comment]
To match osu's default settings, please disable all options in the screenshots above.
The Balance Factor is a feature that already present in the source game, which allows for a certain left and right channel offset based on the x-axis position of the hitobject (similar to 3D sound effects). The larger the value, the more obvious the effect. In general, just keep the default value.

**Reply by miyuki-ts** [reply]
ok ty

**Reply by miyuki-ts** [reply]
i also create 2 issue, idk if the 2nd one is issue, it's that the software does not play skin sounds, like 1, 2, 3, go, section pass, section fail, and the other one i create issue is a issue tho, keyasio randomly crash on some beatmaps, loves and graveyard, but randomly.
